<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageSystemVolumes</class>
 <widget class="QDialog" name="manageSystemVolumes">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>manage system volumes</string>
  </property>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDone">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;done</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="volumePath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFile">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPartition">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>volume path: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>241</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <column>
    <property name="text">
     <string>path to system volumes</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
